libcnb/src/tracing.rs

```diff
@@ -40,45 +49,44 @@ pub(crate) fn start_trace(buildpack: &Buildpack, phase_name: &'static str) -> Bu
     if let Some(parent_dir) = tracing_file_path.parent() {
         let _ = std::fs::create_dir_all(parent_dir);
     }
-    let exporter = match std::fs::File::options()
+
+    let resource = Resource::new([
+        // Define a resource that defines the trace provider.
+        // The buildpac name/version seems to map well to the suggestion here
```

Comment:
```suggestion
        // The buildpack name/version seems to map well to the suggestion here
```

Reply:
Thanks - fixed it myself before I saw this =)